lib/item_repo.rb

```diff
@@ -0,0 +1,22 @@
+require 'csv'
+
+class ItemRepo
+  attr_reader :item_list
+
+  def initialize(input)
+    make_items(input)
+  end
+
+  def make_items(input)
+    items = CSV.open(input, headers: true,
+    header_converters: :symbol)
+
+    @item_list = items.map do |item|
```

Comment:
Do we need to have the instance variable for item list here? Or can we access the reader method like we do on line 20?